r-package/grf/R/join.R

```diff
@@ -0,0 +1,62 @@
+#' Concatenates a list of identical forests that were grown using the same data. 
```

Comment:
A couple comments:
- I think the file should have the same name as the method.
- Don't mean to nitpick, but maybe we could say 'combined' or 'merged together'? Concatenated usually refers to chaining items together into a list.
- Until we've figured out how to merge forests trained on different partitions of the dataset (point 2 in https://github.com/grf-labs/grf/issues/272#issuecomment-445524760), maybe we can hold off on saying "Useful for training forests in a distributed manner"?